test/acceptance/pages.py

```diff
@@ -215,12 +218,30 @@ def assess(self, options_selected):
         >>> page.assess([0, 2, 1])
 
         """
-        for criterion_num, option_num in enumerate(options_selected):
+        def selector(criterion_num, option_num):
             sel = ".rubric_{criterion_num}_{option_num}".format(
                 criterion_num=criterion_num,
                 option_num=option_num
             )
-            self.q(css=self._bounded_selector(sel)).first.click()
+            return sel
+
+        def select_criterion():
+            for criterion_num, option_num in enumerate(options_selected):
+                sel = selector(criterion_num, option_num)
+                self.q(css=self._bounded_selector(sel)).first.click()
+
+        def criterion_selected():
+            for criterion_num, option_num in enumerate(options_selected):
+                sel = selector(criterion_num, option_num)
+                if not self.q(css=self._bounded_selector(sel))[0].is_selected():
+                    return False
+            return True
+
+        attempts = 0
+        while not criterion_selected() and attempts < 5:
```

Comment:
ORA tests don't use the flaky decorator-- I'm not sure if they can. The acceptance tests have some retry logic (there is a retry decorator defined in tests.py), but the accessibility tests haven't used it.

My main concern about doing a flaky/retry thing for these tests is that the 2 particular tests that include training are EXTREMELY long and slow. There is a lot of logging in as different users and going back and forth to make it through the whole workflow. If a failure is hit in training, starting over from the beginning will add quite a bit of time. Also, since the failure typically happens at the same point (the beginning of the second training exercise), starting all over is less likely to be successful vs. re-clicking on the assessment that is already being shown.